### PR TITLE
bubble sort added

### PR DESCRIPTION
**Pull request by manildayma**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- typescript
- 

#### Files Added:
- bubble_sort.ts
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by manildayma** [comment]
@shreyakapoor08  @singh-shreya6  please verify

File: Bubble_Sort/bubble_sort.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//Bubble Sort In Typescript
+
+function bubble(array : number[]) : number[]
+{
+     // print original array
+     console.log("Original Array : " + array);
+     console.log("Steps in bubble sorting the array : ");
+     
+         //FOR LOOP to perform bubble sort(swapping)
+         for( let i = 0; i < array.length ; i++)
```
Comment (MastersAbh) [inline Bubble_Sort/bubble_sort.ts:10]: space after for
no space before let

File: Bubble_Sort/bubble_sort.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+         { 
+              for(let j = 0; j < array.length - 1; j++ ) 
+              {
+                   if(array[j] > array[j + 1])
```
Comment (MastersAbh) [inline Bubble_Sort/bubble_sort.ts:14]: space after if

File: Bubble_Sort/bubble_sort.ts (hunk 1 above)
Comment (manildayma) [reply Bubble_Sort/bubble_sort.ts]: done

File: Bubble_Sort/bubble_sort.ts (hunk 2 above)
Comment (manildayma) [reply Bubble_Sort/bubble_sort.ts]: done

File: Bubble_Sort/bubble_sort.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+     // print bubble sorted array
+     console.log(" Final Array : " + array); 
+     return array;
+}    
+  
+/* consider the array input of 
+bubble sort (may choose any array) */
+bubble([2, 1, 99, 8, 4, 0, 18]);
+
+// OUTPUT
```
Comment (raghav-dalmia) [inline Bubble_Sort/bubble_sort.ts:36]: Add the `sample input` line(instead of output).

File: Bubble_Sort/bubble_sort.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+// [ 1, 2, 8, 99, 4, 0, 18 ]
+// [ 1, 2, 8, 4, 99, 0, 18 ]
+// [ 1, 2, 8, 4, 0, 99, 18 ]
+// [ 1, 2, 8, 4, 0, 18, 99 ]
+// [ 1, 2, 4, 8, 0, 18, 99 ]
+// [ 1, 2, 4, 0, 8, 18, 99 ]
+// [ 1, 2, 0, 4, 8, 18, 99 ]
+// [ 1, 0, 2, 4, 8, 18, 99 ]
+// [ 0, 1, 2, 4, 8, 18, 99 ] 
+// Final Array : 0,1,2,4,8,18,99
```
Comment (raghav-dalmia) [inline Bubble_Sort/bubble_sort.ts:49]: leave a line in EOF

File: Bubble_Sort/bubble_sort.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+// [ 1, 2, 99, 8, 4, 0, 18 ]
+// [ 1, 2, 8, 99, 4, 0, 18 ]
+// [ 1, 2, 8, 4, 99, 0, 18 ]
+// [ 1, 2, 8, 4, 0, 99, 18 ]
+// [ 1, 2, 8, 4, 0, 18, 99 ]
+// [ 1, 2, 4, 8, 0, 18, 99 ]
+// [ 1, 2, 4, 0, 8, 18, 99 ]
+// [ 1, 2, 0, 4, 8, 18, 99 ]
+// [ 1, 0, 2, 4, 8, 18, 99 ]
+// [ 0, 1, 2, 4, 8, 18, 99 ]
```
Comment (raghav-dalmia) [inline Bubble_Sort/bubble_sort.ts:48]: add a line `output`

**Comment by raghav-dalmia** [comment]
Also, make sure to squash the commits after these changes.

**Comment by somya-kapoor** [comment]
Are you working on it?

File: Bubble_Sort/bubble_sort.ts [hunk 6]
```diff
@@ -0,0 +1,53 @@
+//Bubble Sort In Typescript
+
+function bubble(array : number[]) : number[]
+{
+     // print original array
+     console.log("Original Array : " + array);
+     console.log("Steps in bubble sorting the array : ");
+     
+         //FOR LOOP to perform bubble sort(swapping)
+         for (let i = 0; i < array.length ; i++)
```
Comment (MastersAbh) [inline Bubble_Sort/bubble_sort.ts:10]: Please fix indentation

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR.

**Pull request closed without merging**